### ISSUE TITLE
Changing page / url keeps storage?

### ISSUE THREAD

**Issue by HarryBosh** [grevory/angular-local-storage#275]
Hello,

i'm developing on chrome, on ios ipad

Recently when i change pages the local storage is displayed on the new page, even though the url has changed. is this standard operating procedure?

HArry

**Comment by rajkumarpb** [comment]
Yes it is. If you set the locaStorage for site somesite.com, no matter where you navigate, data will be there till you clear/remove it! Hope this helps.

**Comment by HarryBosh** [comment]
So how would one preserve each tabs variables?

**Comment by g2graman** [comment]
@HarryBosh As @rajkumarpb said, storage with this library is meant to be persistent, and that kind of persistence depends on the storage type. I'm not sure I follow your question though. What exactly are you trying to do?

**Comment by rajkumarpb** [comment]
@HarryBosh What do you mean by "Preserve each tab variables?". You don't have to worry about that because data is already in localStorage. If possible, please explain your scenario so we can have faint idea of what you are trying to achieve.
